### PR TITLE
hotfix/11 - attributes with identical names

### PR DESCRIPTION
**Pull request by Munin33**
without this PR the frontend breaks under certain circumstances.

### CODE REVIEW

File: src/views/CreateType.vue [hunk 1]
```diff
@@ -282,8 +282,30 @@ export default class CreateType extends Mixins(Translate) {
...
+      return (this.editingAttribute.name === attribute.name);
+    }).length;
+
+    if (numberOfUses < 2) {
+      if (this.editingAttribute.name) {
+        this.setEponymousAttribute();
+        this.currentlyEditingAttribute = -1;
+      } else {
+        this.$notifications.create({
+        title: `Please choose a name for the attribute.`,
```
Comment (MaximilianV) [inline src/views/CreateType.vue]: Wrong indentation!!!!!

Comment (MaximilianV) [inline src/views/CreateType.vue]: Please use the normal ticks :)

File: src/views/CreateType.vue [hunk 2]
```diff
@@ -282,8 +282,30 @@ export default class CreateType extends Mixins(Translate) {
...
+        details: '',
+        level: NotificationLevel.Critical,
+        timestamp: new Date(),
+        });
+      }
+    } else {
+      this.$notifications.create({
+        title: `There is already an attribute with the choosen name, please choose an other name.`,
```
Comment (MaximilianV) [inline src/views/CreateType.vue]: Please use the normal ticks 💥

**Review by MaximilianV — CHANGES_REQUESTED** [review]

File: src/views/CreateType.vue [hunk 3]
```diff
@@ -282,8 +282,30 @@ export default class CreateType extends Mixins(Translate) {
   }
 
   public saveAttribute(): void {
-    this.setEponymousAttribute();
-    this.currentlyEditingAttribute = -1;
+    const numberOfUses = this.newResourceType.attributes.filter( (attribute) => {
+      return (this.editingAttribute.name === attribute.name);
+    }).length;
+
+    if (numberOfUses < 2) {
```
Comment (friedow) [inline src/views/CreateType.vue]: Please use a guard clause instead.

**Review by friedow — CHANGES_REQUESTED** [review]

**Review by MaximilianV — APPROVED** [review]
:+1:

**Review by friedow — APPROVED** [review]

**Pull request merged**